service/viewer/manager.go

```diff
@@ -134,13 +160,30 @@ func (vm *Manager) RemoveViewer(key interface{}) {
 	})
 }
 
-// 5分に１回呼び出される。チャンネルマップのお掃除
+// 5分に１回呼び出される。チャンネルマップとユーザーマップのお掃除
```

Comment:
```suggestion
// 5分に1回呼び出される。チャンネルマップとユーザーマップのお掃除
```
めっちゃどうでもいいけど、1だけ全角になってそう